Launcher.py

```diff
@@ -338,21 +340,35 @@ def _refresh_components(self, type_filter: Sequence[str | Type] | None = None) -
             scroll_percent = self.button_layout.convert_distance_to_scroll(0, top)
             self.button_layout.scroll_y = max(0, min(1, scroll_percent[1]))
 
-        def filter_clients(self, caller):
+        def filter_clients_by_type(self, caller: MDButton):
```

Comment:
This should probably *also* filter by name if the search box is non-empty? Or clear the search box? Either feels valid.

Reply:
Oh, I just realized: if you start typing with "Adjuster" active, it'll switch to "All" for the search, so clearing the search box is probably the better choice if (as long as) we keep that behaviour.